learningobjects/models.py

```diff
@@ -32,12 +32,12 @@ class LearningObject(models.Model):
     path_xml = models.TextField()
     mpath = models.TextField()
```

Comment:
I think it's highly likely that we might need MPTT, but I don't know yet. It should not block this PR though.  The same for renaming the field. I think I suggested that , but not a blocker.